chainer/sequential.py

```diff
@@ -2,11 +2,11 @@
 import functools
 import inspect
 
+import chainer
 from chainer import function
-from chainer import link
 
 
-class Sequential(link.ChainList):
+class Sequential(chainer.ChainList):
```

Comment:
This change relies on that `chainer.sequential` is imported later than `chainer.link.ChainList` in `chainer/__init__.py`.

Reply:
Thanks, I updated the code by replacing `chainer.*` with `chainer.link.*`. Is it what you meant to fix?

Reply:
I don't think it fixes.
`chainer.link` might not be imported.

Reply:
@toslunar @niboshi Could you check this again?